services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HakemusUpdateRequestFactory.kt

```diff
@@ -25,4 +37,79 @@ object HakemusUpdateRequestFactory {
             representativeWithContacts = null
         )
     }
+
+    fun createFilledJohtoselvityshakemusUpdateRequest(): JohtoselvityshakemusUpdateRequest {
+        return JohtoselvityshakemusUpdateRequest(
+            name = "Testihakemus",
+            postalAddress = PostalAddressRequest(StreetAddress("Kotikatu 1")),
+            constructionWork = true,
+            maintenanceWork = false,
+            propertyConnectivity = false,
+            emergencyWork = false,
+            rockExcavation = false,
+            workDescription = "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+            startTime = ZonedDateTime.now(TZ_UTC),
+            endTime = ZonedDateTime.ofInstant(Instant.now().plus(5, ChronoUnit.DAYS), TZ_UTC),
+            areas = listOf(ApplicationArea("Hankealue 1", GeometriaFactory.polygon)),
+            customerWithContacts =
+                CustomerWithContactsRequest(
```

Comment:
This could use `createCustomerWithContactsRequest`.

Reply:
Changed, thanks!